### PR TITLE
feat: add pipx

### PR DESCRIPTION
**Pull request by meowmeowmeowcat**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

Add spec for [pipx](https://github.com/pypa/pipx)

### CODE REVIEW

File: src/pipx.ts [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+        },
+        {
+          name: "--verbose",
+          description: "Show verbose output",
+        },
+      ],
+      args: {
+        name: "package_spec",
+        description: "Package name or pip installation spec",
+      },
```
Comment (SeparateRecords) [inline src/pipx.ts:66]: You can import the suggestions from `./pip` and add them to this arg!

File: src/pipx.ts [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+          name: "--verbose",
+          description: "Show verbose output",
+        },
+      ],
+      args: [
+        {
+          name: "package",
+          description:
+            "Name of the existing pipx-managed Virtual Environment to inject into",
+        },
```
Comment (SeparateRecords) [inline src/pipx.ts:120]: Same thing as above 🙂

File: src/pipx.ts [hunk 3]
```diff
@@ -0,0 +1,444 @@
...
+          },
+        },
+        {
+          name: "--verbose",
+          description: "Show verbose output",
+        },
+      ],
+      args: {
+        name: "package",
+      },
```
Comment (SeparateRecords) [inline src/pipx.ts:174]: For this and other args where you're specifying a package already installed, you can use a `generator` (https://fig.io/docs/reference/generator) and `pipx list` I think

**Review by SeparateRecords — CHANGES_REQUESTED** [review]
Looking good! Not much to change here. Sorry for the delay too 😅

File: src/pipx.ts (hunk 1 above)
Comment (meowmeowmeowcat) [reply src/pipx.ts:66]: pipx is used to install Python console applications, so not all Python packages can be installed by pipx.

Comment (SeparateRecords) [reply src/pipx.ts:66]: That's fair. Will leave this up to your judgement in that case!

**Pull request merged**